src/app/data/enum/armor-stat.ts

```diff
@@ -104,7 +104,7 @@ export enum SpecialArmorStat {
 
 export enum ArmorPerkOrSlot {
   None,
-  PerkExhumedExcess,
+  PerkAscendantProtector,
```

Comment:
Hi, please note that we have to add enum values at the end of the enum, always.
Otherwise saved configurations will be invalid.

I will fix this manually after the merge.